.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: go
 go:
-  - 1.19.2
+  - 1.22.3
 
 group: bluezone
-dist: trusty
+dist: bionic
```

Comment:
Please add the reason for moving from `trusty` to `bionic`.

Reply:
Trusty spawns travis environment on ubuntu 14. which is coming with older version on builtin gcc and gclib. that was giving error with building s3fs binary and is not compatible with 1.22 go version. 

So I have updated to bionic, this will spawn ubuntu 18 travis environment.